classes/author/Collector.php

```diff
@@ -41,14 +42,16 @@ public function getQueryBuilder(): Builder
     {
         $q = parent::getQueryBuilder();
 
-        $q->when($this->chapterIds !== null, function ($query) {
-            $query->whereIn('author_id', function ($query) {
-                return $query->select('author_id')
-                    ->from('submission_chapter_authors')
-                    ->whereIn('chapter_id', $this->chapterIds);
+        $q->when($this->chapterIds !== null, function (Builder $query) {
+            $query->join('submission_chapter_authors as sca', function (JoinClause $join) {
+                $join->whereIn('sca.chapter_id', $this->chapterIds)
+                    ->whereColumn('a.author_id', 'sca.author_id');
```

Comment:
It seems it work like this too, but usually join would have ON part/statement, i.e. this would be `->on('a.author_id', '=', 'sca.author_id')`